### ISSUE TITLE
[APP SUBMITTED]: 'dict' object has no attribute 'decode'

### ISSUE THREAD

**Issue by ThePieMan** [SickRage/SickRage#1227]
### INFO
Python Version: **2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-8-6.2.9200**
Locale: cp1252
Branch: **master**
Commit: SickRage/SickRage@1abc630b78dbf54b63f5699dc0eeaaf7a6ba4443
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-75805 :: [BTN] :: [1abc630] Exception while searching BTN. Error: AttributeError("'dict' object has no attribute 'decode'",)
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by danfranks** [comment]
Here is a little more information.  All BTN searches are failing for me.

AA
AAAttributeError: 'dict' object has no attribute 'decode'
AA    (search_params.decode('utf-8')), logger.DEBUG)
AA  File "/mnt/mpathj/gingerrustler/sickrage/sickbeard/providers/btn.py", line 87, in search
AA    items_list += self.search(search_string, ep_obj=episode)
AA  File "/mnt/mpathj/gingerrustler/sickrage/sickrage/providers/GenericProvider.py", line 146, in find_search_results
AA    searchResults = curProvider.find_search_results(show, episodes, search_mode, manualSearch, downCurQuality)
AA  File "/mnt/mpathj/gingerrustler/sickrage/sickbeard/search.py", line 497, in searchProviders

**Comment by miigotu** [comment]
Fixed in #1230
